core/components/molecules/pagination-toolbar/pagination-toolbar.md

```diff
@@ -0,0 +1,71 @@
+```meta
+  category: Layout
+  description: "Paginate resource lists and tables"
+```
+
+A `<PaginationToolbar />` component is meant to be used along `ResourceList` and `Table`.
+
+```jsx
+<PaginationToolbar {props} defaults={{items: 20372, perPage: 10, page: 3, showTotals: false}} />
+```
+
+## Example
+
+```js
+class PaginatedResource extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = { page: 1, perPage: 15, items: 7500 }
+  }
+
+  onPageChange(page) {
+    this.setState({ page })
+  }
+
+  render() {
+    const { page, perPage, items } = this.state
+    return (
+      <PaginationToolbar
+        page={page}
+        perPage={perPage}
+        items={items}
+        onPageChanged={this.onPageChange.bind(this)}
+      />
+    )
+  }
+}
+```
+
+### Example with totals
+
+You may want to show a little bit more detail of the content you are
+displaying, so we let you show the totals for the current page by passing the `showTotals`
+prop to the `<PaginationToolbar>` component.
```

Comment:
Not sure if that's the best name, but I can't think of a better one either 😅

`showIndex`,` showLegend`

@landitus how would you describe this component?